environment.yml

```diff
@@ -0,0 +1,105 @@
+name: 2D_strain
```

Comment:
I've been calling this "Strain_2D" now to avoid starting a package name with a number.